cmd/destroy/cmddestroy.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package destroy
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/cmd/util"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"sigs.k8s.io/cli-utils/pkg/apply"
+)
+
+// NewCmdDestroy creates the `destroy` command
+func NewCmdDestroy(f util.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	applier := apply.NewApplier(f, ioStreams)
+	applier.PruneOptions.Destroy = true
```

Comment:
This doesn't seem like a good way to configure it. In particular setting fields multiple levels down.

Reply:
Yeah. This is something I wanted to ask. Is applier a right place to add this field? Like applier.Destroy ?

Reply:
We may not need a `Destroy` field anywhere.